python/Ganga/Runtime/GPIexport.py

```diff
@@ -18,6 +18,31 @@
 from Ganga.GPIDev.Base.Objects import GangaObject
 from inspect import isclass
 
+def _addToInterface(interface, name, _object):
+
+    if isType(_object, GangaObject):
+        setattr(interface, name, addProxy(_object))
+    elif isclass(_object) and issubclass(_object, GangaObject):
+        setattr(interface, name, getProxyClass(_object))
+    else:
+        setattr(interface, name, _object)
+
+def exportToInterface(myInterface, name, _object, doc_section, docstring=None):
```

Comment:
I said
> "It's not intended that importing `ganga` will populate `Ganga.GPI` but I don't think it's a problem if it does so in the short-term.

to say that in the future, everything should be `ganga` but for now it's not a problem if both are populated.

Once we switch to just `ganga` we will just have to go through and clean up this code.